gui/wxpython/gui_core/forms.py

```diff
@@ -580,7 +580,7 @@ def __init__(
         guisizer.Add(self.notebookpanel, proportion=1, flag=wx.EXPAND)
 
         # status bar
-        status_text = _("Enter parameters for '") + self.task.name + "'"
+        status_text = _("Enter parameters for '") + self.task.name + _("'")
```

Comment:
This needs to be done like the other change you have. Use `_("...{name}...").format(name=...)`. Then, the translator can translate the whole message at once (each `_(...)` is a separate message to translate.

Reply:
Ok. Thank you!